spec/models/form_profile_spec.rb

```diff
@@ -1661,6 +1661,7 @@ def expect_prefilled(form_id)
 
             it 'returns prefilled 21-526EZ' do
               Flipper.disable(ApiProviderFactory::FEATURE_TOGGLE_RATED_DISABILITIES_FOREGROUND)
+              Flipper.disable(:disability_compensation_remove_pciu)
```

Comment:
I'm assuming that without the explicit disabling of this Flipper, it's on by default? Just want to make sure we have other tests with it on, which is what I think you're saying in your PR notes already exists.